test_settings.py

```diff
@@ -61,3 +61,6 @@ def root(*args):
         },
     }
 ]
+
+USE_TZ = True
+LEARNING_PATHS_ALLOW_SELF_UNENROLLMENT = True
```

Comment:
Nit: we should use the default behavior for testing (i.e., this should be `False` by default in tests). Also, do we need this? Aren't `learning_paths/settings.py` used in tests?

Reply:
I set this to false. And it looks like we need this as it is set as the `DJANGO_SETTINGS_MODULE` in `manage.py`.